syncer/syncer_test.go

```diff
@@ -841,9 +843,9 @@ func (s *testSyncerSuite) TestTimezone(c *C) {
 		s.resetBinlogSyncer()
 
 		// we should not use `sql.DB.Exec` to do query which depends on session variables
-		// because `sql.DB.Exec` will choose a underlying Conn for every query from the connection pool
-		// and different Conn using different session
-		// ref: `sql.DB.Conn`
+		// because `sql.DB.Exec` will choose a underlying BaseConn for every query from the connection pool
+		// and different BaseConn using different session
+		// ref: `sql.DB.BaseConn`
```

Comment:
these three lines should still be `Conn` not `BaseConn`.

Reply:
I change some comments in syncer_test, you can check it now